src/Utilities/axiosInterceptors.js

```diff
@@ -6,20 +6,6 @@ import { ReadOnlyNotification } from './constants';
 
 const axiosInstance = axios.create();
 
-export async function  accessCheckInterceptor (config)  {
```

Comment:
Removing this interceptor causes some fetches now to be called, and the promises that these fetches are returning are not awaited for. This is the reason for the failing tests. 

The resolution to this is likely going to be to inspect all store actions that do async fetches and make sure they are properly awaited for. Maybe there is an easier way, but we might need to dig a bit more for that.